doc/source/timeseries.rst

```diff
@@ -2372,6 +2372,7 @@ can be controlled by the ``nonexistent`` argument. The following options are ava
 * ``shift``: Shifts nonexistent times forward to the closest real time
 
 .. ipython:: python
+
```

Comment:
the still fails, since `date_range` isn't imported.

Fix in a followup so I can work on 
https://github.com/pandas-dev/pandas/pull/22743?

Reply:
Or feel free to do it here with a `[ci skip]` (and `***NO_CI***` for azure)

Reply:
added a commit here